### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by jankapunkt**
This PR implements the documentation as described in #6 

Implementation progress:

- [x] basic description of this repo

- [x] installation guide

- [x] Contribution guide

- [x] Folder structure

- [x] Style guide

- [x] Testing guide

- [x] CI and deployment guide

- [x] badges :sunglasses:

### CODE REVIEW

**Comment by jankapunkt** [comment]
@ImkeMey @tushar13293 I have now completed the `README` file so far. Please go to the `Files changed` tab and look for the `README.md`, that has no other words in the name.

There are other `README.md` files but they are part of the subfolders and are not subject of review right now.

File: README.md [hunk 1]
```diff
@@ -1,6 +1,194 @@
...
+
+Every contribution is appreciated and we treat every issue and pull request with care.
+Please note, that we follow a certain workflow for contributions.
+
+### Contribution workflow
+
+1. **Create issue**
+
+   There should always be an issue, describing a bug or a feature request before contributing code.
+   Pull requests without issues will be on stale until an issue is created and passed step 2.
```
Comment (tushar13293) [inline README.md]: I wonder if stale should be highlighted?.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,194 @@
...
+   Issues that have did not went through step 2 will get the `needs approval` label.
```
Comment (tushar13293) [inline README.md]: Issues that did not go through step 2 will get the `needs approval` label.

File: README.md [hunk 3]
```diff
@@ -1,6 +1,194 @@
...
+      - [**icon**](imports/ui/components/icon)
+      - [**loading**](imports/ui/components/loading)
+      - [**onloaded**](imports/ui/components/onloaded)
+    - [**pages**](imports/ui/pages)
+      - [**notfound**](imports/ui/pages/notfound)
+    - [**utils**](imports/ui/utils)
+- [**resources**](resources) - Static resources
+  - [**i18n**](resources/i18n) - Translation files
+- [**server**](server)
+- [**tests**](tests) - Top-level of the test suite. ALl tests file are in here
```
Comment (tushar13293) [inline README.md]: ALI (should it be all?) tests file are in here.

File: README.md (hunk 2 above)
Comment (ImkeMey) [inline README.md]: orthography: - have 
(@jankapunkt Rechtschreibfehler ist nicht so wichtig, ist eher zum Testen, ob ich das jetzt richtig mache :-) )

File: README.md [hunk 4]
```diff
@@ -1,6 +1,194 @@
...
+
+3. **Open pull request**
+
+   You should always create a pull request to the `develop` branch, unless there is a strong reason to merge
+   into another specific branch. Note that `master` is disabled for non-core members.
+
+   It is possible to reference multiple issues with a single PR as long as it's made clear, which issues
+   are solved by which parts of the PR. Try to avoid to solve multiple issues with a single commit.
+
+   Pull request that have merge conflicts, failed the linter, failed the tests or miss tests for new features / bigger
```
Comment (ImkeMey) [inline README.md]: orthography: requests (+s)

**Comment by jankapunkt** [comment]
@ImkeMey and @tushar13293 I implemented your changes, please re-review the README.md file and approve/deny the changes.

**Comment by jankapunkt** [comment]
@ImkeMey what do you think?

**Pull request merged**